core/windows_and_tabs/window_management.talon

```diff
@@ -17,8 +17,7 @@ snap last [screen]: user.move_window_previous_screen()
 snap screen <number>: user.move_window_to_screen(number)
 snap <user.running_applications> <user.window_snap_position>:
     user.snap_app(running_applications, window_snap_position)
-# <user.running_applications> is here twice to require at least two applications.
-snap <user.window_split_position> <user.running_applications> <user.running_applications>+:
-    user.snap_layout(window_split_position, running_applications_list)
+snap <user.window_layout>: user.snap_layout(window_layout)
```

Comment:
This capture is ambiguous with the one in line 14. Would recommend you remove the other voice command or make it such that either the number or target windows are required in your `user.window_layout` capture.

Reply:
To avoid the ambiguity I'm going to try changing the name to see what you think. Invoking the layouts with no arguments is useful however so I don't want to require an argument.